code/src/java/pcgen/io/ExportHandler.java

```diff
@@ -336,8 +336,10 @@ private void exportCharacterUsingFreemarker(PlayerCharacter aPC, BufferedWriter
 
 			// data-model
 			Map<String, Object> pc = OutputDB.buildDataModel(aPC.getCharID());
+			Map<String, Object> mode = OutputDB.buildModeDataModel();
```

Comment:
Actually headed the direction that SettingsHandler.getGame() is not the way to access the game mode, that game mode information should be in facets under a DatasetID, so that there is a consistent method of access between different "levels" of information (and we get away from static items at runtime)

I have submitted an additional commit to this PR that makes game mode handling in output more self-consistent (since gamemodename was not consistent)